aea/test_tools/test_cases.py

```diff
@@ -146,6 +146,7 @@ def set_config(
             str(value),
             "--type",
             type_,
+            "--aev",
```

Comment:
@8ball030 We use `--aev` by default on the base test class

Reply:
Cool, I think its important we dont set this by default. otherwise that will lead to a bad time.

Reply:
I'll make this as a argument